pytube/version.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-__version__ = "10.4.1"
+__version__ = "10.4.4"
```

Comment:
We don't update the version number in pull requests, this is handled with bump2version after a PR is merged, so please revert this.

Reply:
I see, no problem :)